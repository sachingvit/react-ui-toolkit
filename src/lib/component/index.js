export { Avatar } from './Avatar/Avatar'
export { Badge } from './Badge/Badge'
export { Label } from './Label/Label'
export { LableInput } from './LabelInput/LableInput'
export { NoContent } from './NoContent/NoContent'
export { Typography } from './Typography/Typography'