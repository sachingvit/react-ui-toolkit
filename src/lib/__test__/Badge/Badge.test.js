import { Badge } from './../../component'
import TestRenderer from 'react-test-renderer';

describe('Badge Compoment', () => {

    // let props;

    // beforeEach(() => {
    //     props = {
    //         label: 'test label string',
    //         id: '1'
    //     }
    // })

    it('Default badge without props', () => {
        const component = <Badge>My dummy text</Badge>
        expect(TestRenderer.create(component).toJSON()).toMatchSnapshot()
    })
})