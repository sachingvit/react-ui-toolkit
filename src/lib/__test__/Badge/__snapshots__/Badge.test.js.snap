// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge Compoment Default badge without props 1`] = `
<span
  className="badge primary"
>
  My dummy text
</span>
`;
